packages/@okta/vuepress-theme-prose/layouts/Layout.vue

```diff
@@ -8,23 +8,37 @@
         'page-body': true,
       }"
     >
-      <Breadcrumb v-if="appContext.isInMobileViewport" />
+      <div v-if="appContext.isTreeNavMobileOpen" class="breadcrumb--holder">
+        <Breadcrumb />
+      </div>
       <div class="content" v-if="$page.frontmatter.component">
         <component :is="$page.frontmatter.component" />
       </div>
+
       <div class="content" v-else>
         <div
           :class="{
             'content--container': true,
-            'navigation-only': appContext.isTreeNavMobileOpen
+            'navigation-only': appContext.isTreeNavMobileOpen,
           }"
         >
           <div class="sidebar-area">
             <Sidebar />
           </div>
           <div class="content-area">
-            <PageTitle />
+            <div v-if="!appContext.isInMobileViewport" class="sticky-content">
+              <PageTitle />
+              <StackSelector v-if="$page.hasStackContent" />
+            </div>
+
+            <div v-if="appContext.isInMobileViewport" class="sticky-content">
+              <Breadcrumb />
+              <StackSelector forMobileViewOnly v-if="$page.hasStackContent" />
```

Comment:
Can `forMobileViewOnly` be replaced with media query styles?
In case it is possible `v-if / v-else` block can be moved inside `sticky-content` container.

Reply:
Wondering the same here seems like using `v-if` and media query should allow us to avoid having to pass something like `forMobileViewOnly`.